index.html

```diff
@@ -135,10 +135,21 @@ <h3>Skills</h3>
                     </ul>
                 </section>
                 <section id="Contact">
-                    <div class="container">
+                    <div class="display">
                         <div class="padd breake">
                             <h2>Get started</h2>
                             <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p> 
+                            <form class="form" action="https://formspree.io/f/xoqrloon" method="post">
+                                <label for="name">
+                                    Your name
+                                    <input type="text" name="name" id="name" maxlength="50" placeholder="Enter your name" required>
+                                </label>
+                                <label for="email" id="email_label">
+                                    Your email
+                                    <input type="email" name="email" id="email" placeholder="Enter your email address" required>
+                                </label>
+                                <textarea id="message" name="message" placeholder="Enter your message here..." maxlength="500" required></textarea>
+                            </form>
                             <button type="button" class="rectangle">Start collaboration</button>             
```

Comment:
- [ ] This form button is outside the form element. Kindly move this button into the element, just before the `</form>` enclosing tag.

- [ ] Your button as it would not submit your form because of the **`type`** attribute. Kindly change button type attribute to `'submit'`, instead of `'button'`

- [ ] Kindly ensure you implement the pointer cursor, hover and pressed (active) effects as demonstrated in the Figma design.

  For the mouse pointer visual, you can do this: `.rectangle {cursor: pointer}`

  ![willy-hover](https://user-images.githubusercontent.com/36542986/160587995-f6b49fc2-28c2-4ea6-939c-8cf83917c684.jpg)